var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -54,6 +54,9 @@ class Paraview(CMakePackage, CudaPackage):
     conflicts('+python', when='@5.6:')
     conflicts('+python3', when='@:5.5')
     conflicts('+shared', when='+cuda')
+    # Legacy rendering dropped in 5.5
+    # See commit: https://gitlab.kitware.com/paraview/paraview/-/commit/798d328c
+    conflicts('~opengl2', when='@5.5:')
```

Comment:
Can we switch the default to be `~opengl2` so that the default spec doesn't fail to concretize?

Reply:
@adamjstewart I'm not getting a failure for concretizing `spack spec paraview`

Is that what you mean by the default spec?

Reply:
Ah sorry, I was mistaken. Most conflicts are for `+feature`, so I thought the default should be True.